media-libs/oidn/oidn-1.4.3-r1.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2023 Gentoo Authors
```

Comment:
Any reason to keep -r0? I'd say this is a candidate for `git mv` style change.

Reply:
No reason other than that I'm not really sure if there are any rules from when to do `git mv` and when not to do it.
So I went for the safe route of keeping the old ebuild around.

Reply:
Note to self: Someone pushed an updated oidn build today:
https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=801be406612d47050cb9a3a64b6a9b58eee1b586

So I should drop this change.